README.md

```diff
@@ -11,3 +11,22 @@ See the [README in the `sdformat_urdf` package](./sdformat_urdf/README.md) for m
   * provides a library and a `urdf_parser_plugin` using that library to convert SDFormat XML to URDF C++ DOM structures
 * [`sdformat_test_files`](./sdformat_test_files/README.md)
   * provides SDFormat models using different parts of the SDFormat XML specification for testing
+
+## Version combinations
+
+This package can be compiled against different Gazebo versions.
+
+Set the `GZ_VERSION` environment variable to the Gazebo version you'd
+like to compile against. For example:
```

Comment:
```suggestion
Set the `GZ_VERSION` environment variable to match the libSDFormat version you'd like to compile against.
For example:
```

Reply:
Nit, I prefer single sentence per line with no line wrapping. I think it makes documentation diffs easier to review.